.idea/inspectionProfiles/profiles_settings.xml

```diff
@@ -0,0 +1,6 @@
+<component name="InspectionProjectProfileManager">
+  <settings>
+    <option name="USE_PROJECT_PROFILE" value="false" />
```

Comment:
The value for the 'USE_PROJECT_PROFILE' option should be 'true' instead of 'false' on line 3.